### PR TITLE
Log improvements

### PR DESCRIPTION
**Pull request by ebroder**

### CODE REVIEW

File: imports/server/configureLogger.ts [hunk 1]
```diff
@@ -26,8 +28,15 @@ logger.format = format.combine(
     level, label, message, [userIdSymbol]: userId, error,
     ...rest
   }) => {
+    const ctx: string[] = [];
+    if (workersCount > 1) {
```
Comment (zarvox) [inline imports/server/configureLogger.ts:32]: I wouldn't be sad about including this regardless

**Review by zarvox — APPROVED** [review]

File: imports/server/configureLogger.ts (hunk 1 above)
Comment (ebroder) [reply imports/server/configureLogger.ts:32]: Eh. It feels like it's just noisy in local dev

File: imports/server/loadBalance.ts [hunk 2]
```diff
@@ -39,14 +39,16 @@ const getWorkersCount = () => {
...
+
 WebApp.onListening(() => {
-  const workersCount = getWorkersCount();
   if (workersCount > 0) {
     http.globalAgent.maxSockets = 99999;
     const proxy = HttpProxy.createProxyServer({
       xfwd: true,
     });
-    const workers = WorkerPool.create(workersCount);
+    workers = WorkerPool.create(workersCount);
```
Comment (zarvox) [inline imports/server/loadBalance.ts:51]: not sure why this is no longer `const workers`?

Comment (ebroder) [reply imports/server/loadBalance.ts:51]: Oh whoops, wilfix

**Pull request merged**